extensions/amp-web-push/0.1/amp-web-push.css

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+amp-web-push.invisible {
+  visibility: hidden;
+}
+
+amp-web-push button {
+  border-radius: 2px;
+  border: 0;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 10px 15px;
+  cursor: pointer;
+  outline: none;
+  font-size: inherit;
```

Comment:
This CSS is now removed because we think it's better to provide styling as an example instead of as a default.